app/views/simple_smart_answers/flow.html.erb

```diff
@@ -2,44 +2,47 @@
   <meta name="robots" content="noindex" />
 <% end %>
 
-<main id="content" class="govuk-grid-column-two-thirds">
-  <section class="simple-smart-answer__question-and-outcome">
-    <% if @flow_state.current_node.outcome? %>
-      <%= render partial: "outcome", object: @flow_state.current_node %>
-    <% else %>
-      <%= render partial: "current_question", locals: {
-        question: @flow_state.current_node,
-      } %>
-    <% end %>
-  </section>
-  <% if @flow_state.completed_questions.any? %>
-    <section>
-      <%= render "govuk_publishing_components/components/heading", {
-        font_size: "m",
-        heading_level: 2,
-        margin_bottom: 4,
-        text: t("formats.simple_smart_answer.your_answers"),
-      } %>
+<main id="content" class="govuk-main-wrapper">
```

Comment:
Our [skip link component](https://components.publishing.service.gov.uk/component-guide/skip_link) looks for `#main-content`. Additionally I think `#content` has some legacy style implications.

Reply:
At the moment the [public layout component sets the skip link to link to `#content`](https://github.com/alphagov/govuk_publishing_components/blob/3b15e3c69698e2f18d95e9e59e2f998dad537779/app/views/govuk_publishing_components/components/_layout_for_public.html.erb#L66), probably for legacy reasons.

Reply:
In this instance `#content` seems apt 🤔 
- thinking we should perhaps align these on the "skip link component"?

Reply:
The default choice on the skip link comes from [the DS page template](https://design-system.service.gov.uk/styles/page-template/) references `#main-content` as part of their default page setup. In my opinion, we ideally want ot be following them as much as possible.

I'd strongly recommend we remove `#content` from  the layout component. That however is a big job and probably there's some big hooks to detach hidden away in our apps, so it's not a job for this PR.

Reply:
> we ideally want to be following them as much as possible.

Agreed - however I _think_ the same discrepancy is happening here too? 
https://design-system.service.gov.uk/components/skip-link/ with this being `#content` being used. 

There might be historical context for this?

Reply:
I saw that and thought about pointing it out as well, however I reckon that's a documentation choice/misstep on the design system team's part.

Reply:
>That however is a big job and probably there's some big hooks to detach hidden away in our apps, so it's not a job for this PR. 

Sure, I've [captured it here via an issue](https://github.com/alphagov/frontend/issues/2790) now.